upload/lambdas/api_server/v1/area.py

```diff
@@ -68,7 +68,9 @@ def schedule_file_validation(upload_area_id: str, filename: str, json_request_bo
     file = upload_area.uploaded_file(urllib.parse.unquote(filename))
     body = json.loads(json_request_body)
     environment = body['environment'] if 'environment' in body else {}
-    validation_id = ValidationScheduler(file).schedule_validation(body['validator_image'], environment)
+    validation_scheduler = ValidationScheduler(file)
+    validation_scheduler.check_file_can_be_validated()
```

Comment:
"Check" leads me to believe that this function returns a boolean. I would consider changing the output of 'check_file_can_be_validated' to be True/False depending on size and then let the api call determine whether to throw exception or continue on with scheduling validation based on output

Reply:
👍